atom/browser/api/atom_api_system_preferences_mac.mm

```diff
@@ -156,29 +171,6 @@ AVMediaType ParseMediaType(const std::string& media_type) {
   DoUnsubscribeNotification(request_id, kNSWorkspaceNotificationCenter);
 }
 
-void SystemPreferences::DoPostNotification(
-    const std::string& name,
-    const base::DictionaryValue& user_info,
-    NotificationCenterKind kind) {
-  NSNotificationCenter* center;
-  switch (kind) {
-    case kNSDistributedNotificationCenter:
-      center = [NSDistributedNotificationCenter defaultCenter];
-      break;
-    case kNSNotificationCenter:
-      center = [NSNotificationCenter defaultCenter];
-      break;
-    case kNSWorkspaceNotificationCenter:
-      center = [[NSWorkspace sharedWorkspace] notificationCenter];
-      break;
-    default:
-      break;
-  }
-  [center postNotificationName:base::SysUTF8ToNSString(name)
-                        object:nil
-                      userInfo:DictionaryValueToNSDictionary(user_info)];
-}
-
```

Comment:
This function declaration should be removed from `atom_api_system_preferences.h` too